index.html

```diff
@@ -1,320 +1,473 @@
+          <div class="details-div display-flex">
+            <div class="cards-header display-flex">
+              <h3 class="style">Multi-Post Stories</h3>
+            </div>
+            <div class="card-info">
+              <p class="style">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+            </div>
+            <ul class="languages-div">
+              <li class="lang style">css</li>
+              <li class="lang style">html</li>
+              <li class="lang style">bootstrap</li>
+              <li class="lang style ruby">Ruby</li>
+            </ul>
+            <div class="btn-div">
+              <button type="button" class="see-project">
+                See Project
+              </button>
+            </div>
           </div>
-          <ul class="languages-div">
-            <li class="lang style">css</li>
-            <li class="lang style">html</li>
-            <li class="lang style">bootstrap</li>
-            <li class="ruby lang style">Ruby</li>
-          </ul>
-          <div class="btn-div">
-            <button type="button" class="see-project">See Project</button>
+        </article>
+      </section>
+      <section class="work-container">
+        <article class="cards-container reverse-div display-flex">
+          <div class="placeholder"></div>
+          <div class="details-div display-flex">
+            <div class="cards-header display-flex">
+              <h3 class="style">Multi-Post Stories</h3>
+            </div>
+            <div class="card-info">
+              <p class="style">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+            </div>
+            <ul class="languages-div">
+              <li class="lang style">css</li>
+              <li class="lang style">html</li>
+              <li class="lang style">bootstrap</li>
+              <li class="lang style ruby">Ruby</li>
+            </ul>
+            <div class="btn-div">
+              <button type="button" class="see-project">
+                See Project
+              </button>
+            </div>
           </div>
+        </article>
+      </section> -->
+    </main>
+
+    <!-- popup window section-->
+    <section id="modal-container" class="container">
+      <div class="pop-up-container display-flex" id="modal-content">
+        <div class="title-container display-flex style">
+          <h2>Multi-Post Stories</h2>
+          <img
+            class="closing-button"
+            src="Images/Closing-Icon.png"
+            alt="Closing Icon"
```

Comment:
- [ ] As previously requested, remove/delete all projects from the index.html file. You only have to use JavaScript dynamically to display the projects.

Reply:
Thank you @mke2111, I have removed the modal section from the HTML file and it is being displayed dynamically from javascript. I also stored the data for the modal in an object and linked them dynamically to their respective sections.